src/routes/users.ts

```diff
@@ -0,0 +1,39 @@
+import express from 'express';
+
+import uploadMiddleware from '../middlewares/handleImage';
+
+import {
+  userValidatorCreate,
+  userValidatorDelete,
+  userValidatorUpdate
+} from '../validators/users';
+import {
+  getUsers,
+  getUser,
+  updateUser,
+  deleteUser,
+  createUser,
+  uploadUserImage
+} from '../controllers/users';
+
+const router = express.Router();
+
+router.get('/users', getUsers);
```

Comment:
the param should show consistency in all the endpoints, so in this case is it better if we choose to work with the param /users and for all those routes we are working with a specific item we can understand that thanks to the id param /users/:id